homeassistant/components/schlage/config_flow.py

```diff
@@ -16,43 +18,97 @@
 STEP_USER_DATA_SCHEMA = vol.Schema(
     {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
 )
+STEP_REAUTH_DATA_SCHEMA = vol.Schema({vol.Required(CONF_PASSWORD): str})
```

Comment:
Could use the same schema I guess and even allow to change the username?

Reply:
I'm not sure changing the username makes sense? That would be a different account with different locks, and it would change the unique id associated with the entry (since the user id would change).

Reply:
The reason I asked was because I wasn't sure the user_id is the e-mail address or not so it could then be possible for the user to actually change his e-mail address. But I guess that's not the case so that's fine.